hub/integrations/tensorflow.py

```diff
@@ -0,0 +1,32 @@
+from hub.util.exceptions import ModuleNotInstalledException
+
+
+def dataset_to_tensorflow(dataset):
+    """Converts the dataset into a tensorflow compatible format"""
+    global tf
+    try:
+        import tensorflow as tf
+    except ModuleNotFoundError:
+        raise ModuleNotInstalledException(
+            "'tensorflow' should be installed to convert the Dataset into tensorflow format"
+        )
+
+    def tf_gen():
+        for index in range(len(dataset)):
```

Comment:
tests are running but this doesn't get covered by codecov for some reason. We had a similar issue with Hub v1. This can be ignored (unless someone knows how to fix this).

Reply:
Is it because tensorflow is imported and that's why you don't see the error?

Reply:
Have you tried running this under no @requires_tensorflow and see if it gets covered?

Reply:
@AbhinavTuli best guess is that codecov is running on simple, not complex, so the coverage from the tensorflow tests aren't getting counted. Haven't checked this, but that's my top suspect.

Reply:
Yeah, that's definitely possible @benchislett.
@davidbuniat I was referring to lines 15-20 and not 9-10.